bamboo/select-stack.js

```diff
@@ -0,0 +1,47 @@
+/* eslint no-console: "off" */
+
+'use strict';
+
+const git = require('simple-git');
+
+function determineIntegrationTestStackName(cb) {
+  const branch = process.env.BRANCH
+
+  if (!branch) return cb('none');
+
+  // Nightly cron job
+  //if (process.env.TRAVIS_EVENT_TYPE === 'cron') return cb('cumulus-nightly');
+
+  if (branch === 'master') return cb('cumulus-from-source');
+
+  const stacks = {
```

Comment:
Have we verified on Bamboo that the author comes through this way and that these will work?